test/svg.jl

```diff
@@ -43,6 +43,42 @@ end
     rm("images/0000000001.png")
 end
 
+@testset "latex alignment in function" begin
+    function latex_ground(args...)
+        background("white")
+        sethue("black")
+    end
+
+    video = Video(400, 200)
+    Background(1:1, latex_ground)
+    Object((args...) -> latex(L"8", O + Point(20, 20), :bottom, :right))
+    Object((args...) -> latex(L"8", O + Point(20, 20), :bottom, :left))
+    Object((args...) -> latex(L"8", O + Point(20, 20), :top, :right))
+    Object((args...) -> latex(L"8", O + Point(20, 20), :left, :top))
+    Object((args...) -> latex(L"8", 0, 0, :middle, :centre))
+    Object(
+        (args...) -> Javis.animate_latex(L"8", O - Point(20, 20), 2, :top, :left, :stroke),
+    )
+    Object(
+        (args...) ->
+            Javis.animate_latex(L"8", O - Point(20, 20), 2, :bottom, :left, :stroke),
+    )
+    Object(
+        (args...) -> Javis.animate_latex(L"8", O - Point(20, 20), 2, :top, :right, :stroke),
+    )
+    Object(
+        (args...) ->
+            Javis.animate_latex(L"8", O - Point(20, 20), 2, :bottom, :right, :stroke),
+    )
+    Object(
+        (args...) ->
+            Javis.animate_latex(L"8", O - Point(-20, 20), 0, :middle, :center, :stroke),
+    )
+    @test_logs (:warn,) (:warn,) render(video; tempdirectory = "images", pathname = "")
```

Comment:
a small comment on why it throws a warning here would be great. Thanks again for your work!